src/__snapshots__/footer.test.jsx.snap

```diff
@@ -207,7 +207,7 @@ exports[`Footer exists 1`] = `
           items={
             Array [
               <span>
-                © 2020 Meetup LLC
+                © 2021 Meetup LLC
```

Comment:
It might make sense to do `new Date().getFullYear()` so we don't have to update this in 2022

Reply:
Oh whoops, nevermind. This is a snapshot